pages/models/generations/SM1/index.md

```diff
@@ -0,0 +1,170 @@
+
+<p align="center">
+<img src="{{'/pages/models/generations/SM1/SM_rot_stat.svg' | relative_url }}"
+     alt="Operation of a Synchronous Machine schematic"
+     style="float: center; width: 400px;" />
+
+</p>
+
+#### Governor
+
+The governor is the controller of the speed of the motor. It will ensure that the frequency of the rotation is maintained at the setpoint, which is the grid frequency.
+The control will be done by changing the amount of steam entering in the turbine, increasing or decreasing the rate of steam depending on the error signal. 
+
+### Operational principles
+
+The dynamic equation that governs the system relates the mechanical torque applied by the turbine and the electrical torque applied by the grid[[3]](#3):
+
+<div style="background-color:rgba(0, 0, 0, 0.0470588); text-align:center; vertical-align: middle; padding:4px 0;">
+
+$$J\frac{2}{p_f}  \frac{d\omega}{dt} = T_m - T_e - T_{fw}$$
+</div>
+
+where $$J$$ is the inertia of the rotating body, $$T_m$$ is the mechanical torque, $$T_e$$ is the electrical torque and $$T_{fw}$$ is the friction and windage torque.
+
+Regarding the electrical part of the model, the rotor winding is magnetized by the exciter current, creating a magnetic field that will rotate with the rotor's angular velocity. As it spins, the stator windings will be subjected to a variable magnetic flux caused by the rotation of the produced magnetic field which, according to Faraday's Law, will induce a current in the windings.
+
+The rotational speed of the rotor of a syncrhonous machine is proportional to the grid frequency times the number of poles (i.e. number of different three-phase windings). The induced field $$e_f$$ will lead the terminal voltage $$v_t$$ by a load angle denoted as $$\delta$$.
+
+<div style="background-color:rgba(0, 0, 0, 0.0470588); text-align:center; vertical-align: middle; padding:4px 0;">
+$$n = \frac{120f}{p_f}$$
+
+</div>
+
+where $$n$$ is the rotational speed in rev/min, $$f$$ is the stator currents frequency in Hz, and $$p_f$$ is the number of field poles.
+
+The equivalent circuit that describes the synchronous generator is[[2]](#2):
+
+<p align="center">
+<img src="{{'/pages/models/generations/SM1/SM_rot_model.svg' | relative_url }}"
+     alt="Operation of a Synchronous Machine schematic"
+     style="float: center; width: 400px;" />
+
+</p>
+
+<div style="background-color:rgba(0, 0, 0, 0.0470588); text-align:center; vertical-align: middle; padding:4px 0;">
+
+$$v_t = e_f - j i_a X_s - i_a R_a \approx e_f - j i_a X_s$$
```

Comment:
did you put all the equations of the models? the set of equations you've wrote isn't sufficient to solve the system.

Reply:
the full model should be completed now, working on the full per unit transformation description